codegenerator/cli/templates/static/codegen/src/globalState/GlobalState.res

```diff
@@ -650,18 +677,38 @@ let actionReducer = (state: t, action: action) => {
       ),
     )
   | SetCurrentlyProcessing(currentlyProcessingBatch) => ({...state, currentlyProcessingBatch}, [])
-  | SetIsInReorgThreshold(isInReorgThreshold) =>
-    if isInReorgThreshold {
```

Comment:
Just checking we never used to exit reorg threshold right?

Reply:
Yes